cpp/rat.cpp

```diff
@@ -922,16 +921,17 @@ PYBIND11_MODULE(rat_core, m) {
         .def_readwrite("pUnitGamma", &Control::pUnitGamma)
         .def_readwrite("boundHandling", &Control::boundHandling)
         .def_readwrite("adaptPCR", &Control::adaptPCR)
+        .def_readwrite("calcSLD", &Control::calcSLD)
```

Comment:
please remove calcSLD from Controls, I think you can safely revert all changes in this file except for line 335

i.e. ` control_struct.calcSLD = false;`